sycl/test-e2e/Matrix/joint_matrix_apply_two_matrices.cpp

```diff
@@ -0,0 +1,14 @@
+//==------ joint_matrix_apply_two_matrices.cpp  - DPC++ joint_matrix--------==//
```

Comment:
if we use the same code on all supported platforms, why do we need to split test to hpp and cpp?
Or do you plan to add sub-group size = 32 version?

Reply:
I should add a test for SG size = 32 like other tests. Thank you for spotting that.